addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -259,8 +259,10 @@ const ColorPaletteWidget = Widget.extend({
             const isCCName = weUtils.isColorCombinationName(colorName);
             if (isCCName) {
                 $color.find('.o_we_cc_preview_wrapper').addClass(`o_cc o_cc${colorName}`);
-            } else {
+            } else if (weUtils.EDITABLE_CSS_VARIABLES.includes(colorName) || compatibilityColorNames.includes(colorName)) {
```

Comment:
If I am not mistaken, each time you use `EDITABLE_CSS_VARIABLES`, you also use `compatibilityColorNames`. Should not the `compatibilityColorNames` odoo module also adds its value in `EDITABLE_CSS_VARIABLES` directly? And maybe find a better name than `EDITABLE_CSS_VARIABLES`?